cdtoolchainv2/cd_toolchain_v2_test.go

```diff
@@ -3007,9 +3016,105 @@ var _ = Describe(`CdToolchainV2`, func() {
+			model.Content = map[string]interface{}{"anyKey": "anyValue"}
+
+			b, err := json.Marshal(model)
+			Expect(err).To(BeNil())
+
+			var raw map[string]json.RawMessage
+			err = json.Unmarshal(b, &raw)
+			Expect(err).To(BeNil())
+
+			var result *cdtoolchainv2.ToolchainEventPrototypeDataApplicationJSON
+			err = cdtoolchainv2.UnmarshalToolchainEventPrototypeDataApplicationJSON(raw, &result)
+			Expect(err).To(BeNil())
+			Expect(result).ToNot(BeNil())
+			Expect(result).To(Equal(model))
+		})
+		It(`Invoke UnmarshalToolchainEventPrototypeDataTextPlain successfully`, func() {
+			// Construct an instance of the model.
+			model := new(cdtoolchainv2.ToolchainEventPrototypeDataTextPlain)
+			model.Content = core.StringPtr("This event is dispatched because the pipeline failed")
+
+			b, err := json.Marshal(model)
+			Expect(err).To(BeNil())
+
+			var raw map[string]json.RawMessage
+			err = json.Unmarshal(b, &raw)
+			Expect(err).To(BeNil())
+
+			var result *cdtoolchainv2.ToolchainEventPrototypeDataTextPlain
+			err = cdtoolchainv2.UnmarshalToolchainEventPrototypeDataTextPlain(raw, &result)
+			Expect(err).To(BeNil())
+			Expect(result).ToNot(BeNil())
+			Expect(result).To(Equal(model))
+		})
+		It(`Invoke UnmarshalToolchainPrototypePatch successfully`, func() {
+			// Construct an instance of the model.
+			model := new(cdtoolchainv2.ToolchainPrototypePatch)
+			model.Name = core.StringPtr("newToolchainName")
+			model.Description = core.StringPtr("New toolchain description")
+
+			b, err := json.Marshal(model)
+			Expect(err).To(BeNil())
+
+			var raw map[string]json.RawMessage
+			err = json.Unmarshal(b, &raw)
+			Expect(err).To(BeNil())
+
+			var result *cdtoolchainv2.ToolchainPrototypePatch
+			err = cdtoolchainv2.UnmarshalToolchainPrototypePatch(raw, &result)
+			Expect(err).To(BeNil())
+			Expect(result).ToNot(BeNil())
+			Expect(result).To(Equal(model))
+		})
+		It(`Invoke UnmarshalToolchainToolPrototypePatch successfully`, func() {
+			// Construct an instance of the model.
+			model := new(cdtoolchainv2.ToolchainToolPrototypePatch)
+			model.Name = core.StringPtr("MyTool")
+			model.ToolTypeID = core.StringPtr("draservicebroker")
+			model.Parameters = map[string]interface{}{"anyKey": "anyValue"}
+
+			b, err := json.Marshal(model)
+			Expect(err).To(BeNil())
+
+			var raw map[string]json.RawMessage
+			err = json.Unmarshal(b, &raw)
+			Expect(err).To(BeNil())
+
+			var result *cdtoolchainv2.ToolchainToolPrototypePatch
+			err = cdtoolchainv2.UnmarshalToolchainToolPrototypePatch(raw, &result)
+			Expect(err).To(BeNil())
+			Expect(result).ToNot(BeNil())
+			Expect(result).To(Equal(model))
+		})
+	})
 	Describe(`Utility function tests`, func() {
 		It(`Invoke CreateMockByteArray() successfully`, func() {
-			mockByteArray := CreateMockByteArray("This is a test")
+			mockByteArray := CreateMockByteArray("VGhpcyBpcyBhIHRlc3Qgb2YgdGhlIGVtZXJnZW5jeSBicm9hZGNhc3Qgc3lzdGVt")
```

Comment:
You need to re-run the secrets checker to update the baseline, otherwise this string will be flagged as a secret. I got caught out for this in one of my recent changes
```
detect-secrets scan --update .secrets.baseline
detect-secrets audit .secrets.baseline
```